README.md

```diff
@@ -1,24 +1,54 @@
 Australian Government Design System Starter Pack
 ==========================
 
-> Get up and running with examples of the design system's components in action and some great `npm` scripts
+> Get started with the Australian Government Design System
 
 
 ## Contents
 
-* [Install](#install)
+* [Get started](#get-started)
+* [How it works](#how-it-works)
 * [Contributing](#contributing)
 * [License](#license)
 
 
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
 
-## Install
+## Get started
 
-Download, clone, or fork this repository and run `npm install` to install all dependencies.
+The starter pack uses NPM (node package manager) to set up a local development environment and install the Design System components. NPM is a package manager for Node based projects. NPM comes pre-installed with Node.
 
-After you have all dependencies installed, run `npm run watch` to spin up a new local server and watch for changes on `*.scss` files. Changes will be injected into your browser so you don’t even need to refresh thanks to [Browser Sync](https://www.browsersync.io/).
+1. Make sure you have `NPM` and `node` installed on your computer. You can follow the [NPM instructions](https://www.npmjs.com/get-npm) if you are unsure.
```

Comment:
npm should be all lower case where applicable. It is also the default package manager for node. It is not an acronym too.